src/Dropdown.php

```diff
@@ -3969,6 +3970,12 @@ public static function getDropdownActors($post, $json = true)
             }
         }
 
+        // permits hooks to alter actors list
+        $results = Plugin::doHookFunction(Hooks::FILTER_ACTORS, [
+            'actors' => $results,
+            'params' => $post,
+        ]);
```

Comment:
1. Most of the time expected returned value from `Plugin::doHookFunction()` has same structure than given param.
2. Hook may return invalid data, it may be a good idea to handle lack of `actors` field in hook result.
3. In your hook example, hook function parameter is named `$params`, and so you access params using `$params['params']`. Maybe here we could change its name to `input`.

```suggestion
        $hook_results = Plugin::doHookFunction(Hooks::FILTER_ACTORS, [
            'actors' => $results,
            'input' => $post,
        ]);

        $results = $hook_results['actors'] ?? [];
```

Reply:
change in escalade according to this one : https://github.com/pluginsGLPI/escalade/commit/ac4f587981a3a7e7b872b3a6543448148462caa0

Reply:
is it ok ?